templates/juniper_junos_show_lldp_neighbors.textfsm

```diff
@@ -0,0 +1,15 @@
+Value Required LOCAL_INTERFACE (\S+)
+Value Required PARENT_INTERFACE (\S+)
+Value Required CHASSIS_ID (\S+)
+Value Required PORT_INFO (\S*)
+Value Required SYSTEM_NAME (\S+)
+
+Start
```

Comment:
I would keep everything under `Start` or act off the complete header row to make sure we're going to capture the correct data.

e.g.
```
Start
  ^Local\s+Interface\s+Parent\s+Interface\s+Chassis\s+Id\s+Port\s+info\s+System\s+Name
  ^blah\s+blah\s+blah

or

Start
  ^Local\s+Interface\s+Parent\s+Interface\s+Chassis\s+Id\s+Port\s+info\s+System\s+Name -> LLDP
```

I hope that makes sense.

Reply:
If you stick with the two separate states, each state requires the following:
```
  ^\s*$$
  ^. -> Error
```

Reply:
Added the complete header and appended 
```
  ^\s*$$
  ^. -> Error
```
to the LLDP state.

Reply:
Condensed everything to one state.